packages/mobile/src/screens/edit-track-screen/components/PublicAvailability.tsx

```diff
@@ -1,13 +1,98 @@
+import { useEffect } from 'react'
+
 import { View } from 'react-native'
 
+import IconVisibilityPublic from 'app/assets/images/iconVisibilityPublic.svg'
 import { Text } from 'app/components/core'
+import { useSetTrackAvailabilityFields } from 'app/hooks/useSetTrackAvailabilityFields'
+import { makeStyles } from 'app/styles'
+import { useColor } from 'app/utils/theme'
+
+const messages = {
+  public: 'Public (Default)',
+  publicSubtitle:
+    'Public tracks are visible to all users and appear throughout Audius.'
+}
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  root: {
+    width: spacing(76)
```

Comment:
This smells weird, I don't know if we've ever used `spacing` with such a large number? Do we need to hardcode this width?

Reply:
yeah.. probably better to do screen dimensions - small spacing